### ISSUE TITLE
Other meta servers not comunicating

### ISSUE THREAD

**Issue by mctrivia** [DigiByte-Core/DigiAssets-Metadata-Server#1]
I set up aws s3 and got a meta server running but any asset i create the meta data is not visible on digiexplorer.info

Just to check if it was just my setup not doing torrents correctly i copied torrents to another computer running uTorrent and successfully had it seeding and other people able to download torrents but digiexplorer.info never downloads the meta data.

**Comment by digicontributer** [comment]
You'll need to seed that metadata to the digiassets metadata API. I will put some thought into a proxy type feature so this is done automatically.

**Comment by mctrivia** [comment]
Yes that would be great since all meta data servers need to be able to
communicate with each other.  They should not be relying on 1 master server.
